pkg/network/time_converter_windows.go

```diff
@@ -0,0 +1,42 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Portions of this code are taken from the gopsutil project
+// https://github.com/shirou/gopsutil .  This code is licensed under the New BSD License
+// copyright WAKAYAMA Shirou, and the gopsutil contributors
+
+package network
+
+import (
+	"time"
+
+	"golang.org/x/sys/windows"
+)
+
+var (
+	modkernel          = windows.NewLazyDLL("kernel32.dll")
```

Comment:
`#include <namingrequest.h>` but could you make this modKernel32 so that we establish a consistent pattern to refer to lazy loaded dll modules?

Reply:
done.